parse_emails/handle_msg.py

```diff
@@ -775,6 +775,18 @@ def _get_canonical_property_name(dir_entry_name):
 
         return None
 
+    @staticmethod
+    def _simplify_text_for_rtf_parsing(rtf_body):
+        """
+            Replaces control characters within the text with their RTF encoded versions \\'HH.
+
+            This function is a temporary hack - should be removed when version 0.0.3 of the RTFDE.deencapsulate package
```

Comment:
please include a link to the original comment where this was suggested in the comments here

Reply:
Added - thanks.